### DISCUSSION TITLE
markdown显示测试

### DISCUSSION THREAD

**Discussion by stars52371** [share121/inter-knot#5313] [其他]
# 标题1
### 标题3

>引用语法测试
>>嵌套引用语法测试

一段包含了*斜体*，**粗体**与***粗斜体***的语句

    code代码块测试

---

链接测试[Markdown语法教学](https://markdown.com.cn)

**Comment by stars52371** [comment]
# 测试结果
---
能表现出正确的类型，但是因背景颜色导致了引用格式的边线和代码块的背景看不出来

**Comment by aomengxin** [comment]
感谢大佬！！！
![1726635086500](https://github.com/user-attachments/assets/406c41bd-869b-4319-9e87-e0b4b300b7f5)
